src/blocks/AnimatedCtaBlock.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react'
+import styled from 'react-emotion'
+import {
+  CONTENT_GUTTER,
+  ContentWrapper,
+  MOBILE_BP_DOWN,
+  SectionWrapper,
+  TABLET_BP_DOWN,
+} from '../components/blockHelpers'
+import { LazyLottie } from '../components/LazyLottie'
+import { BaseBlockProps, MarkdownHtmlComponent } from './BaseBlockProps'
+
+const AnimatedCtaWrapper = styled('div')({
+  display: 'flex',
+  textAlign: 'center',
+  flexWrap: 'wrap',
+  minWidth: '100%',
+})
+
+const AnimatedCta = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  marginRight: 'auto',
+  textAlign: 'center',
+  width: `calc(33.333333% - 1.333333333rem)`,
+  '&:not(:nth-of-type(3n))': {
+    marginRight: CONTENT_GUTTER,
+  },
+  [TABLET_BP_DOWN]: {
+    width: 'calc(50% - 1rem)',
+    '&:not(:nth-of-type(3n))': {
+      marginRight: 0,
+    },
+    '&:not(:nth-of-type(2n))': {
+      marginRight: CONTENT_GUTTER,
+    },
+  },
+  [MOBILE_BP_DOWN]: {
+    width: '100%',
+    '&:not(:nth-of-type(2n))': {
+      marginRight: 0,
+    },
+  },
+})
+
+const SectionTitle = styled('h2')({
+  fontSize: '3.75rem',
+  width: '100%',
+  [TABLET_BP_DOWN]: {
+    fontSize: '2.25rem',
+    marginTop: '3rem',
+  },
+})
+
+const AnimatedCtaHead = styled('div')({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  maxWidth: '13.125rem',
+  marginLeft: 'auto',
+  marginRight: 'auto',
+})
+
+const AnimatedCtaBody = styled('div')()
+
+const AnimatedCtaTitle = styled('h3')({
+  fontSize: '1.375rem',
+  margin: 0,
+})
+
+const AnimatedCtaParagraph = styled('div')({
+  fontSize: '1.125rem',
```

Comment:
i'm not saying this particular one is wrong, but i'm concerned that we effectively use `16px` font sizes in most places and `18px` in _some_ places. Feels weird?